Makefile

```diff
@@ -68,7 +68,7 @@ integration: cmd/algorand-indexer/algorand-indexer
 	test/postgres_integration_test.sh
 
 e2e: cmd/algorand-indexer/algorand-indexer
-	cd misc && docker-compose build --build-arg GO_IMAGE=${GO_IMAGE} && docker-compose up --exit-code-from e2e
+	cd misc && docker-compose build --build-arg GO_IMAGE=${GO_IMAGE} && VERSION=${CI_E2E_FILE_VERSION} docker-compose up --exit-code-from e2e
```

Comment:
I wonder if `VERSION` or `CI_E2E_FILE_VERSION` can just be added to the `e2e` service in `misc/docker-compose.yml`. The more we can unify the locations of similar env vars, the easier it will be to comprehend the meaning. But I haven't tried to make sure it works.